docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,221 @@
+# Create Your First Tizen IoT .NET Application
+
+The Tizen .NET framework allows you to easily and efficiently create applications for Tizen. Study the following instructions to help familiarize yourself with the Tizen .NET application development process. With the instructions, you can create and run a basic .NET application, which displays some texts on the screen without any user interaction.
+
+1.  Before you get started with developing Tizen applications, set up the [development environment](../../../vstools/install.md).
+
+2. [Create a Project](#create-a-project) using Visual Studio.
+
+    This step shows how you can use a predesigned project template that creates all the basic files and folders required for your project.
+
+3. [Build Your Application](#build-your-application).
+
+    After you have implemented code for the features you want, this step shows how you can build the application to validate and compile
+    the code.
+
+4. [Deploy and Run Your Application](#deploy-and-run-your-application).
+
+    This step shows how you can deploy and run the application on the reference board.
+
+5. [Understand Your Application](#understand-your-application).
+
+    This step shows the default structure of the basic files.
```

Comment:
Thanks for the update, @xbldx

@nawab-samsung I also hope you check the sentence and let us know your opinion.

Reply:
@mijin85cho 
I feel to align with the content under heading **Understand Your Application**, I suggest slight modification. 
```suggestion
This step shows the controls used in the application and explains the default structure of the basic files.
```
Request your opinion on this and if you find this to be fine, we can request @xbldx to update it.

Reply:
@mijin85cho
If you agree his suggestion, I will apply it. Please let me know it.

Reply:
@xbldx thanks for your cooperation and active participation in clearing our doubts.

Reply:
I think @nawab-samsung's enhancement  is acceptable.
Adding to that, I suggest the sentence rewriting like below.

This step explains the controls used in the application and the default structure of the basic files.

Reply:
@mijin85cho seems fine, request @xbldx to kindly update the sentence. Following which I will check the file thoroughly again and approve in case all is well.

Reply:
Fixed it.